ui/src/components/Form/DynamicForm.vue

```diff
@@ -37,7 +35,7 @@
         :codes="field.options"
         :ref="field.name"
         :hint="field.short"
-        :value="field.value"
+        :value="selectedUseNameCode!=''? selectedUseNameCode:field.value"
```

Comment:
Same idea as the above. Set the default value from the add person page and let it propagate from there.